Samples/word-blazor-add-in/word-blazor-add-in/Pages/Index.razor.js

```diff
@@ -0,0 +1,245 @@
+﻿/* Copyright(c) Maarten van Stam.All rights reserved.Licensed under the MIT License. */
+
+/**
+ * Step 1: Add some Paragraphs to the document
+ */
+export async function setupDocument() {
+
+    await Word.run(async (context) => {
+
+        context.document.body.clear();
+        context.document.body.insertParagraph("One more paragraph. ", "Start");
+        context.document.body.insertParagraph("Co-locating Index.razor.js Demo", "Start");
+        context.document.body.insertParagraph("Inserting another paragraph. ", "Start");
+
+        context.document.body.insertParagraph(
+            "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
```

Comment:
```suggestion
            "Effective add-ins offer unique and compelling functionality that extends Office applications in a visually appealing way. To create a great add-in, provide an engaging first-time experience for your users, design a first-class UI experience, and optimize your add-in's performance.",
```
Replacing with our own text from docs (the previous was from an existing web site. )

Reply:
Fixed by earlier commit